client/index.css

```diff
@@ -199,3 +199,119 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    position: absolute;
+    top: 40px;
+    left: 0;
+    right: 0;
+    width: 345px;
+    height: auto;
+    margin: auto;
+}
+#settings #icon-username {
+    position: absolute;
+    top: 30px;
+    left: 100px;
+    width: 100px;
+    height: auto;
+}
+#icon-username h2 {
+    position: absolute;
```

Comment:
Don't use absolute here. Use a `float: left` for the image, [clearfix](https://css-tricks.com/snippets/css/clear-fix/), and appropriate margins.